napari_animation/_qt/keyframeslist_widget.py

```diff
@@ -1,180 +1,41 @@
-from qtpy.QtCore import QSize
-from qtpy.QtGui import QIcon, QImage, QPixmap
-from qtpy.QtWidgets import QListWidget, QListWidgetItem
+from napari._qt.containers import QtListModel, QtListView
+from qtpy.QtCore import QModelIndex, QSize, Qt
+from qtpy.QtGui import QImage
 
 
-class KeyFramesListWidget(QListWidget):
-    """List of key frames for an animation
+class KeyFrameModel(QtListModel):
+    """Model for QtListView of KeyFrames"""
```

Comment:
I don't have any particular convention to point to here :) I'd say whatever feels right.  I put them in the same file here since they're both small, and likely to always be used together.